src/entities/payment/api/payment.ts

```
@@ -1,11 +1,17 @@
 import axiosInstance from '@/shared/api/axios';
-import { PaymentList } from '@/entities/payment/model/payment.type';
+import { PaymentActionResult, PaymentList } from '@/entities/payment/model/payment.type';
```

ESLint

[error] Replace `·PaymentActionResult,·PaymentList·` with `⏎··PaymentActionResult,⏎··PaymentList,⏎`

(prettier/prettier)